src/modules/constants.ts

```diff
@@ -208,6 +254,9 @@ export const GNOSIS_SAFE_NETWORKS: NetworkInfo[] = [
   BINANCE_NETWORK,
   OPTIMISM_NETWORK,
   POLYGON_NETWORK,
+  GOERLI_TEST_NETWORK,
+  RINKEBY_TEST_NETWORK,
+  VOLTA_TEST_NETWORK,
```

Comment:
We can get rid of these. Too many testnets doesn't really add any value. We can just have Rinkeby, for example.